specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -195,7 +195,7 @@
               "$ref": "#/definitions/DiskPool"
             }
           },
-          "202": {
+          "201": {
```

Comment:
As per this spec, x-ms-long-running-operation is set to true for PUT request which means its a long running operation. Can you specify the x-ms-long-running-operation-options as well to let the callers know the long operation pattern.
Refer http://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options

Also doesn't your service return 202 status now?

Reply:
@RamyasreeChakka fixed this.

Yes, at the time we were returning 202, but now the server should be returning 201.